crates/ruff_python_stdlib/src/typing.rs

```diff
@@ -267,6 +272,11 @@ pub fn is_pep_593_generic_member(member: &str) -> bool {
     matches!(member, "Annotated")
 }
 
+/// Returns `true` if a name matches that of the `Literal` generic.
+pub fn is_literal_member(member: &str) -> bool {
+    matches!(member, "Literal")
+}
```

Comment:
These are obviously sort of dumb but they exist for consistency with the other variants (e.g., `is_pep_593_generic_member`).